src/pages/tasks/NewTaskPage.js

```diff
@@ -82,7 +82,7 @@ function NewTaskPage(props) {
             if (!assigneeDetails) {
                 return setErrorMessage(props.translate('task.assigneeError'));
             }
-            const displayDetails = Task.getAssignee(assigneeDetails);
+            const displayDetails = Task.getAssignee(assigneeDetails, props.personalDetails);
```

Comment:
We are doing something unnecessary computing for `assigneeDetails` let's remove that here. 
And just pass `assigneeAccountID` & personalDetails for the getAssignee method